### PR TITLE
Feat/documentation

### PR DESCRIPTION
**Pull request by tonymike99**
I have created the documentation site for my CL.
Still, some refactoring and changes have to be done.
https://thirsty-mclean-8a0078.netlify.app/index.html

### CODE REVIEW

File: assets/css/components.css [hunk 1]
```diff
@@ -1,49 +1,50 @@
...
 section {
-    /* border: 3px solid black; */
     margin: 1rem;
 }
 
 .container {
-    margin: 1rem;
+    border: 1px solid lightgrey;
+    border-radius: 5px;
+    margin: 0.5rem;
```
Comment (itsmebipul00) [inline assets/css/components.css]: margin and margin bottom again, you could use one

**Comment by itsmebipul00** [comment]
Great work!!

**Review by itsmebipul00 — APPROVED** [review]

File: Components/Button/button.css [hunk 2]
```diff
@@ -136,3 +131,7 @@
     border: 1px solid var(--dark-color);
     color: var(--dark-color);
 }
+
+.btn-floating {
+    padding: 10px 0.7rem;
```
Comment (zoef-yt) [inline Components/Button/button.css]: use px for 0.7 rem

File: assets/css/components.css [hunk 3]
```diff
@@ -1,49 +1,50 @@
...
+    margin-bottom: 3rem;
+    padding: 0.5rem;
```
Comment (zoef-yt) [inline assets/css/components.css]: use px for less than 1rem

**Review by zoef-yt — CHANGES_REQUESTED** [review]
Some places you used rem for value less than 1 use px there

File: assets/css/components.css (hunk 1 above)
Comment (tonymike99) [reply assets/css/components.css]: I have changed it. Thanks Bipul :)

File: assets/css/components.css (hunk 3 above)
Comment (tonymike99) [reply assets/css/components.css]: I have made the changes. Thanks for point them out, Zoef :)

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Pull request merged**